libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKResourceUtils.m

```diff
@@ -52,10 +52,16 @@ + (NSString *)localizedString:(NSString *)localizationKey
 
 + (UIImage *)imageNamed:(NSString *)name
 {
-    NSBundle *bundle = [NSBundle bundleForClass:[self class]];
     NSAssert(name != nil, @"name must contain a value.");
+    //Get image from main bundle (this is to allow extracting SDK Assets when using template app as we include SDK Assets as part of the tempalte app)
```

Comment:
Of course I say that so as to deliberately put my foot in my mouth, and thereby confirm that the solution works. ;-)  I had tried using the `resource_bundles` approach, without luck.  But when I add the following to the SalesforceSDKCore.podspec:
```
sdkcore.resource = 'shared/resources/SalesforceSDKAssets.xcassets'
```
That seems to do the right thing, both with and without `use_frameworks!`.  So it looks like that could be the solution.